docarray/typing/url/url_3d/url_3d.py

```diff
@@ -18,6 +18,10 @@ class Url3D(AnyUrl, ABC):
     Can be remote (web) URL, or a local file path.
     """
 
+    @classmethod
+    def mime_type(cls) -> str:
+        return 'application'
```

Comment:
what is this mime type? use constants alsl

Reply:
This is a broad category for mimetypes. Contains obj, pdf, json, xml and many other files.. Now that I think about it, we should make it more specific (whatever is associated with .obj extension because that's what we usually use) and for other non-obj files rely on extra extensions. Changed accordingly.